CONTRIBUTING.adoc

```diff
@@ -93,7 +93,14 @@ NOTE: Replace "linux/amd64" with the appropriate value for your platform/archite
 
         $ sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
         $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
-        $ oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
+        $ sudo ./oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
```

Comment:
the only files referenced by that command are `openshift.local.config/master/openshift-registry.kubeconfig` and `openshift.local.config/master/admin.kubeconfig` which the two preceding lines should have made readable.